### PR TITLE
1.1 use media file bundle

### PR DESCRIPTION
**Pull request by ngilain**
https://github.com/open-orchestra/open-orchestra/pull/733
https://github.com/open-orchestra/open-orchestra-media-admin-bundle/pull/92
https://github.com/open-orchestra/open-orchestra-media-bundle/pull/138
https://github.com/open-orchestra/open-orchestra-media-file-bundle/pull/2

### CODE REVIEW

**Comment by ghost** [comment]
:+1:

**Comment by ggrasset** [comment]
:+1:

**Comment by alavieille** [comment]
:+1:

File: MediaFileBundle/Manager/UploadedMediaManager.php [hunk 1]
```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace OpenOrchestra\MediaFileBundle\Manager;
+
+use Knp\Bundle\GaufretteBundle\FilesystemMap;
+
+/**
+ * Class UploadedMediaManager
+ */
+class UploadedMediaManager
```
Comment (nicolasThal) [inline MediaFileBundle/Manager/UploadedMediaManager.php:10]: you could have moved this class in the namespace `MediaFile\Manager`.

This way the bundle would be only the link between the code and the framework

**Pull request merged**